docs/beta-scenarios.md

```diff
@@ -0,0 +1,103 @@
+# Beta Testing Connectors in the Extension Gallery  
+-------
+
+Welcome to the Beta of the Connectors in the Extension Gallery. Tableau has worked closely with a number of its Technology Partners to create connectors for their platform. These connector plugins will replace having to connect to your data using the "Other ODBC/JDBC" connector, and provide a smooth and performant connection to these databases. Installing the new connector only takes a few minutes and you'll be able to see the difference for yourself.
```

Comment:
I don't think this page belongs in the SDK documentation. The SDK documentation is for _building_ connectors, not _using_ them. Sending people who just want to download a connector and use it to this site will confuse them.

Instead, you should add this information to the [KB connector plugins page](https://help.tableau.com/current/pro/desktop/en-us/examples_connector_sdk.htm), especially because that is the page you are sent to if you click "Learn more" on the connector gallery and it has no mention of the gallery as of now. You'll have to ask Doralee or another technical writer how to do that.